feeder/feeder.go

```diff
@@ -49,6 +51,12 @@ func loadConfig() (FeederConfig, error) {
 		return config, err
 	}
 
+	for _, w := range config.Whitelist {
+		if strings.Contains(w, ":") {
```

Comment:
This is "dangerous", a valid image like `my-awesome-registry:5000/opensuse/foobar` would be rejected.

I think you can use some code like [that](https://github.com/flavio/kube-image-bouncer/blob/master/rules/not_latest.go) to parse the image into something more meaningful